features/head.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <head>
-description: The `<head>` element TODO.
+description: The `<head>` element contains machine-readable information (metadata) about the document.
```

Comment:
For web developers, isn't scripts and stylesheets the most important things in `<head>`? I'll leave this description to @ddbeck's review.